tools/ci_build/build.py

```diff
@@ -576,6 +576,10 @@ def run_server_tests(build_dir, configs):
     for config in configs:
         config_build_dir = get_config_build_dir(build_dir, config)
         server_app_path = os.path.join(config_build_dir, 'onnxruntime_server')
+        if is_windows():
```

Comment:
In line 575, could you please install numpy and protobuf as well, in case the build environment does not have it?